nav2_dwb_controller/dwb_plugins/src/xy_theta_iterator.cpp

```diff
@@ -104,7 +104,7 @@ bool XYThetaIterator::isValidSpeed(double x, double y, double theta)
   {
     return false;
   }
-  if (vmag_sq == 0.0 && th_it_->getVelocity() == 0.0) {
+  if (vmag_sq == 0.0 && theta == 0.0) {
```

Comment:
@SteveMacenski the function should not use the member but the function parameter. The only place this function is called is in line 115 where `th_it_->getVelocity()` is passed as `theta`.